pkg/antctl/raw/multicluster/get/member_token.go

```diff
@@ -107,41 +103,41 @@ func runEToken(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		memberTokens = append(memberTokens, memberToken)
+		outToken := &memberToken
```

Comment:
I think we should also ignore the token if it has no `common.CreateByAntctlAnnotation`?

Reply:
I feel for getting a single token, no need to check the token is created by antctl or not. Why we do not support getting a token created by other ways?